Doc/library/zipfile.rst

```diff
@@ -130,10 +130,12 @@ ZipFile Objects
 ---------------
 
 
-.. class:: ZipFile(file, mode='r', compression=ZIP_STORED, allowZip64=True)
+.. class:: ZipFile(file, mode='r', compression=ZIP_STORED, allowZip64=True, \
+                   compresslevel=None)
 
    Open a ZIP file, where *file* can be a path to a file (a string), a
    file-like object or a :term:`path-like object`.
+
```

Comment:
Per the [style guide](https://devguide.python.org/documenting/#use-of-whitespace), I took the liberty of adding line breaks here; the constructor explanation was already quite long.